blockchain/blockchain_test.go

```diff
@@ -486,8 +494,7 @@ func TestEvents(t *testing.T) {
 
 		require.NoError(t, filter.SetRangeEndBlockByHash(blockchain.EventFilterFrom,
 			utils.HexToFelt(t, "0x3b43b334f46b921938854ba85ffc890c1b1321f8fd69e7b2961b18b4260de14")))
-		require.NoError(t, filter.SetRangeEndBlockByHash(blockchain.EventFilterTo,
-			utils.HexToFelt(t, "0x3b43b334f46b921938854ba85ffc890c1b1321f8fd69e7b2961b18b4260de14")))
+		require.NoError(t, filter.SetRangeEndBlockByNumber(blockchain.EventFilterTo, 8))
```

Comment:
What is this test saying? Why was the hash removed? 

If the previous test is still valid we should create a new test case

Reply:
same reason in https://github.com/NethermindEth/juno/pull/780#discussion_r1222866023 

we need to include the pending block in the range, cant do that with a hash

Reply:
Unresolving because we discussed adding another test case